tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es5",
+    "outDir": "build",
+    "importHelpers": true,
+    "noImplicitAny": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "preserveConstEnums": true,
+    "pretty": true,
+    "sourceMap": true
```

Comment:
I know the guidelines don't include it as standard (though I've just filed [a PR](https://github.com/resin-io/process/pull/97)), but we should really include `"strictNullChecks": true` here too.